pkgs/os-specific/linux/roccat-tools/default.nix

```diff
@@ -0,0 +1,112 @@
+{ stdenv, fetchurl, cmake, dbus, glib, libcanberra, atk, cairo
+, gdk_pixbuf, gtk2, pango, libnotify, libgudev, xorg, sqlite
+, gegl, dbus_glib, gettext, pkgconfig, shadow, devices ? false
+}:
```

Comment:
Passing devices as a list of names is more user-friendly. Could you do that instead? (Use `concatStringsSep` to process the argument).